src/XamlStyler/StylerService.cs

```diff
@@ -19,18 +21,65 @@ namespace Xavalon.XamlStyler
 {
     public class StylerService
     {
+        private readonly string[] ignoredNamespacesInOrdering = new string[]
+        {
+            "http://schemas.microsoft.com/expression/blend/2008",
+        };
         private readonly DocumentManipulationService documentManipulationService;
-        private readonly Dictionary<XmlNodeType, IDocumentProcessor> documentProcessors;
+        private readonly IStylerOptions options;
         private readonly XmlEscapingService xmlEscapingService;
+        private Dictionary<XmlNodeType, IDocumentProcessor> documentProcessors;
 
         public StylerService(IStylerOptions options)
         {
             this.xmlEscapingService = new XmlEscapingService();
             this.documentManipulationService = new DocumentManipulationService(options);
 
+            this.options = options;
+        }
+
+        /// <summary>
+        /// Execute styling from string input
+        /// </summary>
+        /// <param name="xamlSource"></param>
+        /// <returns></returns>
+        public string StyleDocument(string xamlSource)
+        {
+            string xamlOutput = xamlSource;
+
+            if (this.documentManipulationService.AllowProcessing)
+            {
+                // Escape all xml entity references to ensure that they are output exactly as given.
+                var escapedDocument = this.xmlEscapingService.EscapeDocument(xamlSource);
+
+                // Parse XDocument.
+                var xDocument = XDocument.Parse(escapedDocument, LoadOptions.PreserveWhitespace);
+
+                // Manipulate the document tree.
+                var manipulatedDocument = this.documentManipulationService.ManipulateDocument(xDocument);
+
+                // Find ignored namespaces in document.
+                var ignoredNamespacesLocalNames = FindIgnoredNamespaces(manipulatedDocument);
```

Comment:
nit: this.

Reply:
StylerService.FindIgnoredNamespaces - as the method is static - but added class name.